ProcessLib/Utils/ProcessUtils.cpp

```diff
@@ -43,17 +44,66 @@ std::vector<std::reference_wrapper<ProcessVariable>> findProcessVariables(
     std::vector<ProcessVariable> const& variables,
     BaseLib::ConfigTree const& pv_config,
     std::initializer_list<std::string>
-        tag_names)
+        tags)
```

Comment:
Please, clang format.

Reply:
I checked that too, it is the way how clang-format outputs it.

Reply:
Moving the function signature into one line and then running clang-format will produce what you expect. Once I move into next step to make a commit, the pre-commit hook will tell me that the staged content is not formatted correctly. I would suggest to keep it so far.